kofta/src/vscode-webview/components/RoomCard.tsx

```diff
@@ -49,7 +49,9 @@ export const RoomCard: React.FC<RoomProps> = ({
     <div>
       <button
         onClick={onClick}
-        className={`w-full ${active ? "bg-simple-gray-4d" : "bg-simple-gray-33"} hover:bg-simple-gray-69 active:bg-simple-gray-23 py-2.5 px-5 rounded-lg`}
+        className={`w-full ${
+          active ? "bg-simple-gray-4d" : "bg-simple-gray-33"
```

Comment:
This code reformatting does not change functionality. I think I'd prefer this PR without this change.

Reply:
this is just prettier doing it